### DISCUSSION TITLE
How can I install a new runner?

### DISCUSSION THREAD

**Discussion by pmartiner** [lutris/lutris#3772] [Q&A] [answered]
Hi, everyone! A Lutris newbie here. :)
I recently saw that we're able to install new runners besides Citra, Dolphin, Linux, Steam and Wine. I tried to search how to do it, but I didn't find any info about it. I was wondering if anyone could help me achieve that :)
Thanks!

PS: I'm trying to add Ryujinx as a runner

**Comment by CartoonFan** [comment, marked as answer]
I don't know if you've already found your answer, but if you click on the three horizontal lines near the top of the window, there should be an entry that says "Preferences". If you click on that, there's an entry on the left that says "Runners". From there, just scroll down to the runner you want and click on the box-looking icon (the one that's next to the wrench and screwdriver). Hope that helps!
